Flaky Windows tests

### What happened?

Periodically:

https://github.com/pulumi/pulumi/runs/7023093966?check_suite_focus=true#step:35:21

```
C:\hostedtoolcache\windows\Python\3.9.13\x64\python.exe: can't open file 'D:\a\pulumi\pulumi\pkg\apulumipulumi\scripts\go-test.py': [Errno 2] No such file or directory
mingw32-make: *** [makefile:121: test_pkg_nodejs] Error 2
```

### Steps to reproduce

-

### Expected Behavior

-

### Actual Behavior

-

### Versions used

_No response_

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 PROJECT_NAME := Pulumi SDK
-PROJECT_ROOT := $(realpath .)
+PROJECT_ROOT := .
 SDKS         := dotnet nodejs python go
 SUB_PROJECTS := $(SDKS:%=sdk/%)
 

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
 PROJECT_NAME := Pulumi SDK
-PROJECT_ROOT := .
+PROJECT_ROOT := ..
 SDKS         := dotnet nodejs python go
 SUB_PROJECTS := $(SDKS:%=sdk/%)
 

--- a/sdk/dotnet/Makefile
+++ b/sdk/dotnet/Makefile
@@ -2,7 +2,7 @@ PROJECT_NAME    := Pulumi .NET Core SDK
 LANGHOST_PKG    := github.com/pulumi/pulumi/sdk/v3/dotnet/cmd/pulumi-language-dotnet
 
 PROJECT_PKGS    := $(shell go list ./cmd...)
-PROJECT_ROOT    := $(realpath ../..)
+PROJECT_ROOT    := ../..
 
 DOTNET_VERSION  := $(if ${PULUMI_VERSION},${PULUMI_VERSION},$(shell cd ../../ && pulumictl get version --language dotnet))
 

--- a/sdk/go/Makefile
+++ b/sdk/go/Makefile
@@ -4,7 +4,7 @@ VERSION          := $(if ${PULUMI_VERSION},${PULUMI_VERSION},$(shell cd ../../ &
 TEST_FAST_PKGS   := $(shell go list ./pulumi/... ./pulumi-language-go/... ./common/... | grep -v /vendor/ | grep -v templates)
 TEST_AUTO_PKGS   := $(shell go list ./auto/... | grep -v /vendor/ | grep -v templates)
 TESTPARALLELISM  ?= 10
-PROJECT_ROOT     := $(realpath ../..)
+PROJECT_ROOT     := ../..
 
 include ../../build/common.mk
 

--- a/sdk/nodejs/Makefile
+++ b/sdk/nodejs/Makefile
@@ -3,7 +3,7 @@ NODE_MODULE_NAME := @pulumi/pulumi
 VERSION          := $(if ${PULUMI_VERSION},${PULUMI_VERSION},$(shell cd ../../ && pulumictl get version --language javascript))
 
 LANGUAGE_HOST    := github.com/pulumi/pulumi/sdk/v3/nodejs/cmd/pulumi-language-nodejs
-PROJECT_ROOT     := $(realpath ../..)
+PROJECT_ROOT     := ../..
 
 PROJECT_PKGS    := $(shell go list ./cmd...)
 TESTPARALLELISM ?= 10

--- a/sdk/python/Makefile
+++ b/sdk/python/Makefile
@@ -2,7 +2,7 @@ PROJECT_NAME     := Pulumi Python SDK
 LANGHOST_PKG     := github.com/pulumi/pulumi/sdk/v3/python/cmd/pulumi-language-python
 VERSION          := $(if ${PULUMI_VERSION},${PULUMI_VERSION},$(shell cd ../../ && pulumictl get version))
 PYPI_VERSION 	 := $(if ${PULUMI_VERSION},${PULUMI_VERSION},$(shell cd ../../ && pulumictl get version --language python))
-PROJECT_ROOT     := $(realpath ../..)
+PROJECT_ROOT     := ../..
 
 PYENV := ./env
 PYENVSRC := $(PYENV)/src

